src/lib/updating-element.ts

```diff
@@ -529,22 +545,32 @@ export abstract class UpdatingElement extends HTMLElement {
   /**
    * Updates the element. By default this method reflects property values to attributes.
    * It should be overridden to render and keep updated DOM in the element's root.
-   * Within `update()` setting properties does not trigger `invalidate()`, allowing
-   * property values to be computed and validated before DOM is rendered and updated.
-   * This means in an override of `update()`, before calling `super.update()`
-   * setting properties will not trigger another update, but after calling `super.update()`
-   * setting properties will trigger another update.
+   * Setting properties inside this method will *not* trigger the element to update.
```

Comment:
...will not trigger a *second* round of `update`

Reply:
Changed.